server/game/cards/11.4-MoD/VictarionGreyjoy.js

```diff
@@ -0,0 +1,72 @@
+const DrawCard = require('../../drawcard');
+
+class VictarionGreyjoy extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Kneel a warship location',
+            phase: 'challenge',
+            cost: ability.costs.kneel(card => card.getType() === 'location' && card.hasTrait('warship')),
+            handler: context => {
+                this.context = context;
+
+                this.game.promptWithMenu(this.controller, this, {
+                    activePrompt: {
+                        menuTitle: 'Choose one',
+                        buttons: [
+                            { text: 'Kneel a location', method: 'kneelLocation' },
+                            { text: 'Stand Victarion', method: 'standVictarion' },
+                            { text: 'Gain renown', method: 'gainRenown' }
+                        ]
+                    },
+                    source: this
+                });
+            }
+        });
+    }
+
+    kneelLocation() {
+        this.game.promptForSelect(this.controller, {
+            activePromptTitle: 'Select a location',
+            source: this,
+            cardCondition: card => card.getType() === 'location',
+            gameAction: 'kneel',
+            onSelect: (player, card) => this.onLocationSelected(player, card),
+            onCancel: (player) => this.cancelSelection(player)
+        });
+
+        return true;
+    }
+
+    standVictarion(player) {
+        this.controller.standCard(this);
+
+        this.game.addMessage('{0} uses {1} to stand {1}', player, this);
```

Comment:
Per convention, messages contain costs, so in this case `{0} uses {1} and kneels {2} to stand {1}`, with `{2}` being `this.context.costs.kneel`.